app/app.global.css

```diff
@@ -30,6 +30,7 @@
   --color_thunderbird: #d62f12;
   --color_black_medium_opacity: rgba(0, 0, 0, 0.6);
   --color_black_low_opacity: rgba(0, 0, 0, 0.25);
+  --color_active_green: rgb(9, 255, 0);
```

Comment:
nit: 
- Since this color doesn't have the alpha value, you can just use the hex value (`#09ff00`).
- `active` is unnecessary. Other color names don't include the state of UI like active and disbaled

Reply:
Done.